### PR TITLE
Improve bug report templates

### PR DESCRIPTION
**Pull request by Triticum0**
This is aimed at improving bug reporting. not fully finished not sure how you want the layout to be. have another draft with a slightly different layout on my repo used 
[OBS](https://github.com/obsproject/.github/blob/master/.github/ISSUE_TEMPLATE/bug_report.yaml) and [Pcsx2 ](https://github.com/PCSX2/pcsx2/issues/new?assignees=&labels=Bug&template=emu_bug_report.yaml&title=%5BBUG%5D%3A+) as reference.

### CODE REVIEW

**Comment by Squall-Leonhart** [comment]
Your file has no extension, github will not read it without the .yaml.

**Comment by Triticum0** [comment]
fixed it

**Comment by Zorklis** [comment]
Anything else needed or can this be merged

**Comment by Squall-Leonhart** [comment]
its still got a massive TO DO on it.

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+       | Drivers Verison     | 4.0.0 NVIDIA 512.95      |
+       | Cemu Verison        | 2.0      | 
+      value: |
+       | Info                | Sys Info        | 
+       |---------------------|-----------------|
+       | OS                  | |
+       | CPU                 | |
+       | GPU                 | |
+       | Drivers Verison     | |
+       | Cemu Verison        | |
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:60]: Markdown table is prone to editing errors by users. Would be better as separate fields, or just plain lines (no table).

Also 4 typos of `Verison`

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+labels: [Bug]
+body:
+  - type: markdown
+    id: md_readme
+    attributes:
+      value: |
+        ## Important: Read First
+        
+        This is an issue tracker for tracking bugs and feature requests, and as such, please refrain from using it to submit support requests.
+        If you need any support links are provided in the readme.md
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:14]: Suggest rewriting as:

> This issue tracker is for bug reports and feature requests.
> 
> For tech support or questions about Cemu, [follow the links here](https://github.com/cemu-project/Cemu#links).

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        
+         ### Please Avoid Issues Pertaining to the Following:
+         -TO DO
+  - type: textarea
+    id: current_behavior
+    attributes:
+      label: Current Behavior
+      description: "What the bug is, in a breif decription"
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:22]: Typo `brief description`

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      description: |
+        Any non-default core settings.
+    validations:
+      required: false
+  - type: textarea
+    id: logs_files
+    attributes:
+      label: "Logs"
+      description: |
+        "Log files are located in the cemu folder".
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:74]: Suggest rewriting as:

> Attach `log.txt` from your Cemu folder (*File > Open Cemu folder*)

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# Docs - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a issue with the emulation (ie. graphical issue, crashes, unintended behavior with doesn't occur on hardware).
+title: "[BUG]: "
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:4]: Is having `[BUG]: ` in the title of every bug necessary?

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+labels: [Bug]
```
Comment (jn64) [inline .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:5]: Existing label is `bug`, not `[Bug]`. https://github.com/cemu-project/Cemu/issues?q=label%3Abug

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml (hunk 5 above)
Comment (Triticum0) [reply .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml:4]: probably not, just thought it might be easy to separate issues at glance. reverted.

Comment (jn64) [reply .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml]: @Triticum0 Thanks. Sorry, but you might have to click [*Resolve conversation*](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#resolving-conversations) on each review that you've completed. Apparently the reviewer can't do it unless they have write access to the repo :confused:

To save us some additional button clicking, there are 2 remaining typos after https://github.com/cemu-project/Cemu/pull/47/commits/f8721b3d04c418491cac170b03a7191811f81e7e:

- `decription` -> `description`
- `reprodue` -> `reproduce`

**Comment by Triticum0** [comment]
I fixed the issue you mentioned, thought there are a few things that still need resolving before being merged.

1. I made two versions one of them has a few tweaks, namely separating the system info into individual tabs and adding a save files box check-in [here](prhttps://github.com/Triticum0/Cemu/blob/Bug_report_type_1/.github/ISSUE_TEMPLATE/Emulation_bug_report.yml) if you think anything adds should be added to this pr.
2. Line 12 should be removed.
3. Lines 17-18 should be removed if not needed

File: .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml (hunk 5 above)
Comment (Triticum0) [reply .github/ISSUE_TEMPLATE/Emulation_bug_report.yaml]: Fixed the typos

**Comment by Exzap** [comment]
Took a while but we finally updated the issue templates (ca01e923bf03573d5023feb0f4464ce015910ea6). What we ended up using is derived from your input, thanks!

**Pull request closed without merging**